addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -346,6 +346,12 @@ var MassMailingFieldHtml = FieldHtml.extend({
         var $snippets = $snippetsSideBar.find(".oe_snippet");
         var $snippets_menu = $snippetsSideBar.find("#snippets_menu");
 
+        for (const button of $snippets_menu.get(0).children) {
+            if (button.textContent == 'Select a template') {
```

Comment:
please use `===` instead of `==`
Would have been nice to be able to target the element without having to search it by text content but I looked and indeed there's nothing else to really identify it :grimacing: